cogs5e/sheetManager.py

```diff
@@ -721,15 +728,16 @@ async def send_ddb_ctas(ctx, character):
             inline=False
         )
     # game log
-    if character.ddb_campaign_id and gamelog_flag:
+    if character.ddb_campaign_id and gamelog_flag and show_campaign_cta:
         try:
             await CampaignLink.from_id(ctx.bot.mdb, character.ddb_campaign_id)
         except NoCampaignLink:
             embed.add_field(
                 name="Link Your D&D Beyond Campaign",
                 value=f"Sync rolls between a Discord channel and your D&D Beyond character sheet by linking your "
                       f"campaign! Use `{ctx.prefix}campaign https://www.dndbeyond.com/campaigns/"
-                      f"{character.ddb_campaign_id}` in the Discord channel you want to link it to.",
+                      f"{character.ddb_campaign_id}` in the Discord channel you want to link it to."
+                      f"This message can be disabled in {ctx.prefix}server_settings.",
```

Comment:
This feels like a weird place to advertise turning it off, lol - we can probably do without this message

Reply:
I felt like it would be noticable there, because for most people they would either link a campaign or want the message to stop appearing. Either way they'd only see it once or twice and having the method to disable it right there when it appears seems most convenient. Plus it would make people more aware that disabling it is even an option.